web_api/national_park/national_parks.py

```diff
@@ -1,6 +1,6 @@
 import requests
 import os
-from park import Park
+from .park import Park
```

Comment:
For some reason, this is erroring for me now with this message: ImportError: attempted relative import with no known parent package.

Reply:
thats due to you running this as a standalone application...

Reply:
when you import it it should be good. (like when you import it in the web_api.py file that is controlling them all